surveys/2021-trademark-survey/questions.md

```diff
@@ -111,13 +112,14 @@ trademarks that, without the license, would be an infringing use.
 >
 > TODO
 
-### How much of the Rust language can someone change and still call it “Rust”?
+### How much of the Rust language/toolchain can someone change and still call it “Rust”?
```

Comment:
This question somewhat implicitly assumes a fork of the existing Rust toolchain. In other words, this seems to care about what the diff is between the existing Rust language and tooling and new artifacts that come from the existing ones. I do wonder if we're missing out of subtle distinctions when someone creates an entirely new artifact and calls it Rust. I think this distinction gets at the heart of the fact that we have two groupings of things that are called "Rust": the abstract language definition which isn't really a thing but more an idea, and the actual artifacts that implement that idea (e.g., the compiler, cargo, etc).

Reply:
I was trying to think of ways to reword this question to incorporate what I understand you to be getting at here, but it was tough given the responses we have for this question. I feel it would have to be new question of something like:

> When someone creates a new artifact for the Rust ecosystem, what is required to actually call it Rust?

The hard part here is coming up with the right choices for the survey responses.